src/test/java/jpa/repository/MemberRepositoryTest.java

```diff
@@ -34,4 +46,44 @@ void findByName() {
         List<Member> memberList = memberRepository.findByEmail(expected);
         assertThat(memberList.size()).isEqualTo(1);
     }
+
+    @Test
+    @DisplayName("사용자는 여러 즐겨찾기를 가질 수 있음을 테스트")
+    void findFavoritesByMember() {
+        //given
+        Station station1 = new Station("선릉역");
+        Station station2 = new Station("의정부역");
+        Station station3 = new Station("회룡역");
+        Station station4 = new Station("도봉산역");
+        Station station5 = new Station("삼성역");
+
+        stationRepository.save(station1);
+        stationRepository.save(station2);
+        stationRepository.save(station3);
+        stationRepository.save(station4);
+        stationRepository.save(station5);
```

Comment:
```suggestion
        stationRepository.saveAll(Arrays.asList(station1, station2, station3, station4, station5));
```

repository save에 대한 반환값을 사용하는 테스트가 아니라면 저는 이렇게 한번에 save합니다 :)

Reply:
오오 saveAll 도 있군요!!